### PR TITLE
Parse `rec` in update_lit_checks.py

### PR DESCRIPTION
**Pull request by tlively**

### CODE REVIEW

**Comment by tlively** [comment]
Current dependencies on/for this PR:
* main
  * **PR #4784** <a href="https://app.graphite.dev/github/pr/WebAssembly/binaryen/4784" target="_blank">Graphite</a>  👈
    * **PR #4785** <a href="https://app.graphite.dev/github/pr/WebAssembly/binaryen/4785" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/WebAssembly/binaryen/4784?utm_source=stack-comment).

**Review by kripken — APPROVED** [review]

**Comment by tlively** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/WebAssembly/binaryen/4784?utm_source=stack-comment).


Job Reference: wHPoO07O2vDdfBC3TH7I

**Pull request merged**
